src/pages/DrawerViewPage/DrawerViewPage.vue

```diff
@@ -238,11 +249,15 @@ const errorStore = useErrorStore();
 onMounted(async () => {
   fetchStatus.value = "fetching";
   // get current drawer contents
-  await drawerStore.refreshDrawer(props.drawerId);
+  const drawerRecord = await drawerStore.refreshDrawer(props.drawerId);
+
+  if (!drawerRecord) {
+    return errorStore.setError(new Error("Couldn't find drawer content."));
+  }
 
   // sortBy is persisted in the drawer record, so check its current value
-  const drawerRecord = drawerStore.drawerRecords[props.drawerId];
-  if (!drawerRecord.contents) {
+  // const drawerRecord = drawerStore.drawerRecords[props.drawerId];
```

Comment:
strip?

Reply:
ah, good catch. 

Adding this in while trying to debug a whiltescreen issue. Turned out my login had expired, and the drawer I was viewing wasn't accessible anymore, causing the page to break at `v-if="drawer.contents"`. Added a `v-if="drawer"` on the `<Tabs>`, which let the login prompt trigger.

All this is to say, keep your eye out for issues caused by login expirations.  If more bubble up, we should talk through some strategies for avoiding this.